heudiconv/cli/run.py

```diff
@@ -148,9 +158,11 @@ def get_parser():
                         'DICOMs to anonymized IDs. Such command must take a '
                         'single argument and return a single anonymized ID. '
                         'Also see --conv-outdir')
-    parser.add_argument('-f', '--heuristic', dest='heuristic_file',
-                        required=True,
-                        help='python script containing heuristic')
+    parser.add_argument('-f', '--heuristic', dest='heuristic',
+                        # some commands might not need heuristic
```

Comment:
we can remove the comment and just raise an error if required but not specified

Reply:
or may be we would refactor to always have a command... who knows ;-)  comment was left primarily for myself I guess, since initially it was a required option.